mesop/examples/dict_state.py

```diff
@@ -0,0 +1,30 @@
+import mesop as me
```

Comment:
[question] Should this be some kind of e2e test rather than an example?

Reply:
that's fair, i was being a little lazy :) done.